### DISCUSSION TITLE
"Cancel" button on an extension's settings

### DISCUSSION THREAD

**Discussion by Alwaysin** [FreshRSS/FreshRSS#6860] [General]
I thought the "cancel" button was for "close the slider", whereas it seems that it means "erase unsaved changes". I was opening a new issue when I realized this.

Maybe it could be rephrased, and a close button could be added because it is not always straight-forward to have to click outside the slider to close it. Or the existing button could be "erase unsaved changes AND close the slider".

![chrome_ouf6lhMLD6](https://github.com/user-attachments/assets/efb8783f-c4be-4f94-8b02-8fa4148e3af1)

**Comment by Frenzie** [comment]
That may be a regression. If not the text should read something more like "Reset".

Incidentally you can also press Escape.

> because it is not always straight-forward to have to click outside the slider to close it.

I don't quite follow this one though. It's either everywhere on the left (simple) or on the top if too small.

**Comment by math-GH** [comment]
see here too: https://github.com/FreshRSS/FreshRSS/issues/3662

It is a reset button not a close button

**Reply by Frenzie** [reply]
That explains why it rang a vague bell and I thought it might be a regression. :-)
